modules/rhel-compute-requirements.adoc

```diff
@@ -19,7 +19,7 @@ ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.6 and later] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.8 and later] with "Minimal" installation option.
```

Comment:
```suggestion
** Base operating system: Use link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.8 and later] with the "Minimal" installation option.
```

Reply:
Does "Minimal" here refer to RHEL or OCP? I do not see any RHEL doc that stands out in the link destination.